internal/node/node_loader.js

```diff
@@ -493,17 +492,15 @@ module.constructor._resolveFilename = function(request, parent, isMain, options)
 
 // Before loading anything that might print a stack, install the
 // source-map-support.
-if (INSTALL_SOURCE_MAP_SUPPORT) {
-  try {
-    const sourcemap_support_package = path.resolve(process.cwd(),
-          '../build_bazel_rules_nodejs/third_party/github.com/source-map-support');
-    require(sourcemap_support_package).install();
-  } catch (_) {
-    log_verbose(`WARNING: source-map-support module not installed.
-      Stack traces from languages like TypeScript will point to generated .js files.
-      Set install_source_map_support = False in ${TARGET} to turn off this warning.`);
-  }
+try {
+  const sourcemap_support_package = path.resolve(
```

Comment:
@soldair has pointed out we can remove this from the loader (which we want to delete) by passing the script to the `--require` flag in the launcher.

Reply:
another advantage of the `--require` method is that it'll be passed down to workers and child processes

Reply:
👍